cocotb/share/makefiles/simulators/Makefile.vcs

```diff
@@ -73,11 +73,19 @@ $(SIM_BUILD)/simv: $(SIM_BUILD) $(VERILOG_SOURCES) $(SIM_BUILD)/pli.tab $(COCOTB
 	LD_LIBRARY_PATH=$(LIB_DIR):$(LD_LIBRARY_PATH) TOPLEVEL=$(TOPLEVEL) \
 	$(CMD) -top $(TOPLEVEL) $(PLUSARGS) +acc+1 +vpi -P pli.tab +define+COCOTB_SIM=1 -sverilog $(EXTRA_ARGS) -debug -load libvpi.so $(COMPILE_ARGS) $(VERILOG_SOURCES)
 
+$(SIM_BUILD)/runsim.tcl:
+	@echo "# Autogenerated file" > $@
+ifeq ($(GUI),1)
+	@echo "dump -aggregates -add /" >> $@
+	@echo "run" >> $@
+	@echo "quit" >> $@
+endif
```

Comment:
Why do you only dump signals in gui mode?

Reply:
otherwise there won't be any signal values visible on waveform